.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: c
+sudo: false
+
+cache:
+  directories:
+    - ~/arduino_ide
+
+install:
+  - make setup-ino
```

Comment:
It's a little redundant, but you may want to consider passing the OS here as a form of documentation:

```shell
make setup-ino CI_SYSTEM=$TRAVIS_OS_NAME
```

Reply:
I made a change in the makefile to check the uname to determine MacOS or not. Would this still be useful?